planning/behavior_path_planner/include/behavior_path_planner/scene_module/avoidance/avoidance_module_data.hpp

```diff
@@ -74,6 +77,12 @@ struct AvoidanceParameters
   // continue to detect backward vehicles as avoidance targets until they are this distance away
   double object_check_backward_distance;
 
+  // object's enveloped polygon
+  double object_envelope_buffer;
```

Comment:
can you add default value at `object_envelope_buffer` and in `threshold_time_object_is_moving`?